src/main/kotlin/lotto/model/LottoStatisticFormat.kt

```diff
@@ -0,0 +1,24 @@
+package lotto.model
+
+/**
+ * 로또 당첨번호 비교한 통계를 위한 데이터 클래스
+ * */
+data class LottoStatisticFormat(
+    val purchasedPrice: Price,
+    val winList: HashMap<LottoRank, Int>,
+) {
+    private val totalPrice: Int = LottoRank.values().filter { it != LottoRank.MISS }.sumOf { getRankProfit(it) }
```

Comment:
```suggestion
    private val totalPrice: Int = LottoRank
        .values()
        .filter { it != LottoRank.MISS }
        .sumOf { getRankProfit(it) }
```
함수 체이닝 컨벤션을 적용해주면 좋을 것 같아요